lib/utils.js

```diff
@@ -56,8 +56,10 @@ const buildUrl = (host, port, ssl, endpoint) => {
 };
 
 // Custom Logger
-const log = (msg) => {
-    console.log('[' + new Date().toISOString().substring(2, 19).replace('T', ' ').replace(/[Z-]/g, '') + '] ' + msg);
+const log = (message, level) => {
+    let datetime = new Date();
+    datetime = `${datetime.toISOString().replace('T', ' ').split('.')[0]} ${datetime.toString().split(' ')[5].split('GMT')[1]}`;
+    console.log(`${datetime} [${(level || 'info')}] ${message}`);
```

Comment:
highly recommend just have unit test for this, use [mockdate](https://www.npmjs.com/package/mockdate) library to mock  timezone/date and validate results

Reply:
We'll visit this in the next improvement - integration testing will cover this part - in the next improvement, if needed, whole unit testing stuff is gonna be visited